crates/sui-rest-api/src/checkpoints.rs

```diff
@@ -94,6 +94,106 @@ async fn get_checkpoint_full(
     .pipe(Ok)
 }
 
+pub struct GetCheckpointFullBatch;
+impl ApiEndpoint<RestService> for GetCheckpointFullBatch {
+    fn method(&self) -> axum::http::Method {
+        axum::http::Method::GET
+    }
+
+    fn path(&self) -> &'static str {
+        "/checkpoints/full/batch"
+    }
```

Comment:
We probably just want the path to be "/checkpoints/full" to be consistent with the other apis